packages/typescript/src/TypescriptMutator.ts

```diff
@@ -30,6 +30,10 @@ export class TypescriptMutator {
   }
 
   private mutateForNode<T extends ts.Node>(node: T, sourceFile: ts.SourceFile): Mutant[] {
+    if (shouldNodeBeSkipped(node)) {
+      return [];
+    }
```

Comment:
Nitpicking: Could we put the rest of the method in an `else` expression for clarity?

Reply:
Funny, I prefer the [opposite](https://github.com/palantir/tslint/pull/4502). 😄